scripts/baremetalset_cleanup_stale_entries.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+# USAGE: ./script.sh <ROLE name - case sensitive> <list of compute names to remove>
+#        e.g. ./script.sh Compute compute-2 compute-3
+
```

Comment:
Is it worth `set -eu` and `-o pipefail` to fail fast e.g if curl/jq are not in the $PATH